docs/Protocol/README.md

```diff
@@ -6,6 +6,6 @@ Each protocol document should describe a single API call and the expected
 behavior of both sides of that API call, including common error conditions.
 
 ## Contents
-| Document | API call | Description |
-|:---|---:|:---|
-| publishBlockStream.md | `publishBlockStream` | The communication between a publisher and a block node when publishing a block stream from an authoritative source such as a consensus node.|
+| Document                                       | API call | Description |
+|:-----------------------------------------------|---:|:---|
```

Comment:
Note, it isn't necessary to align the header lines, and trying to maintain those can be a time waste; the previous setup (with minimal headers) avoids that waste.

Note: Many IDE's try to do this automatically; it may be helpful to turn that feature off, if possible.

Reply:
✅ I agree about the maintainability for the tables, changed.